docs/adguard-home/getting-started.md

```diff
@@ -0,0 +1,256 @@
+
+### Windows
+
+1. Open *Control Panel* through Start menu or Windows search.
+
+1. Go to *Network and Internet* category and then to *Network and Sharing Center.*
+
+1. On the left side of the screen find the *Change adapter settings* button and click it.
+
+1. Select your active connection, right-click it and choose *Properties.*
+
+1. Find *Internet Protocol Version 4 (TCP/IPv4)* (or, for IPv6, *Internet Protocol Version 6 (TCP/IPv6)*) in the list, select it and then click Properties again.
+
+1. Choose *Use the following DNS server addresses* and enter your AdGuard Home server addresses.
+
+### macOS
+
+1. Click the Apple icon and go to *System Preferences.*
+
+1. Click *Network.*
+
+1. Select the first connection in your list and click *Advanced.*
+
+1. Select the DNS tab and enter your AdGuard Home server addresses.
+
+### Android
+
+1. From the Android Menu home screen, tap *Settings.*
+
+1. Tap *Wi-Fi* on the menu. The screen listing all of the available networks will be shown (it is impossible to set custom DNS for mobile connection).
+
+1. Long press the network you’re connected to and tap *Modify Network.*
+
+1. On some devices, you may need to check the box for *Advanced* to see further settings. To adjust your Android DNS settings, you will need to switch the IP settings from *DHCP* to *Static.*
+
+1. Change set DNS 1 and DNS 2 values to your AdGuard Home server addresses.
+
+### iOS
+
+1. From the home screen, tap *Settings.*
+
+1. Choose *Wi-Fi* in the left menu (it is impossible to configure DNS for mobile networks).
+
+1. Tap the name of the currently active network.
+
+1. In the *DNS* field enter your AdGuard Home server addresses.
+
+## Running without superuser (Linux only) {#running-without-superuser}
+
+You can run AdGuard Home without superuser privileges, but you need to either grant the binary a capability (on Linux) or instruct it to use a different port (all platforms).
+
+### Granting the necessary capabilities
+
+Using this method requires the `setcap` utility. You may need to install it using your Linux distribution’s package manager.
+
+To allow AdGuard Home running on Linux to listen on port 53 without superuser privileges and bind its DNS servers to a particular interface run:
+
+```sh
+sudo setcap 'CAP_NET_BIND_SERVICE=+eip CAP_NET_RAW=+eip' ./AdGuardHome
+```
+
+Then run `./AdGuardHome` as an unprivileged user.
+
+### Changing the DNS listen port
+
+To configure AdGuard Home to listen on a port that does not require superuser privileges, stop AdGuard Home, open `AdGuardHome.yaml` in your editor, and find these lines:
+
+```yaml
+dns:
+    # …
+    port: 53
+```
+
+You can change the port to anything above 1024 to avoid requiring superuser privileges. If the file does not exist, create it in the same folder, type these two lines down and save.
```

Comment:
```suggestion
You can change the port to anything above 1024 to avoid requiring superuser privileges. If the file does not exist, create it in the same folder, type these two lines down, and save.
```